scripts/remove_old_images.py

```diff
@@ -0,0 +1,74 @@
+import datetime
+import json
+import subprocess
+
+
+def get_untagged_images(image_name: str):
+    image_versions = []
+    page = 1
+    while True:
+        result = subprocess.run(
+            " ".join(
+                [
+                    "gh",
+                    "api",
+                    "-H",
+                    '"Accept: application/vnd.github+json"',
+                    f'"https://api.github.com/orgs/tinkoff-ai/packages/container/etna%2F{image_name}/versions?page={page}"',
+                ]
+            ),
+            shell=True,
+            check=True,
+            stdout=subprocess.PIPE,
+        )
+        parsed_result = json.loads(result.stdout.decode("utf-8"))
+        if len(parsed_result) == 0:
+            break
+        else:
+            image_versions += parsed_result
+            page += 1
+    # filter out tagged images
+    image_versions = [image for image in image_versions if len(image["metadata"]["container"]["tags"]) == 0]
+    return image_versions
+
+
+def get_list_to_remove(leave_last_n_images: int, image_versions: list):
+    image_versions = sorted(
+        image_versions, key=lambda x: datetime.datetime.strptime(x["created_at"], "%Y-%m-%dT%H:%M:%SZ")
+    )
+    return image_versions[:-leave_last_n_images]
+
+
+def remove_images(image_versions: list):
+    for image in image_versions:
+        print(f"Removing {image['url']}")
+        subprocess.run(
```

Comment:
Why do we need such a strange way of running this code?

Reply:
For some reason non-get empty requests return 422 error.
This is workaround.

https://github.com/cli/cli/issues/3937